source/includes/fundamentals/code-snippets/Skip.java

```diff
@@ -0,0 +1,88 @@
+package docs;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Sorts;
+
+public class Skip {
+    
+    private final MongoCollection<Document> collection;
+    private final MongoClient mongoClient;
+    private final MongoDatabase database;
+
+    private Skip() {
+        final String uri = System.getenv("DRIVER_REF_URI");
+
+        mongoClient = MongoClients.create(uri);
+        database = mongoClient.getDatabase("crudOps");
+        collection = database.getCollection("skip");
+        // end declaration
+    }
+
+    public static void main (String [] args){
+        Skip skip = new Skip();
+        // skip.setupPaintCollection();
+        skip.skipExample();
+    }
+
+    private void skipExample(){
+        // begin skipExample
+        import com.mongodb.client.model.Filters;
+        import com.mongodb.client.model.Sorts;
+
+        // <MongoCollection setup code here>
+
+        Bson filter = Filters.empty();
+        collection.find(filter)             
+            .sort(Sorts.descending("qty"))        
```

Comment:
If you want the most popular, those should be the highest quantity, right? If you skipped the first 5 of 8 in descending order, then you would get the bottom 3. Or if you meant qty remaining, then you should mention that in your example description.

Reply:
I mentioned in the intro that the best colors sell first -- so I meant for the example to look for the 3 lowest quantities

Reply:
Ok sorry, I must have skipped a sentence.